website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -31,70 +31,53 @@ import TOCInline from "@theme/TOCInline";
+To get up and running with Weave GitOps Enterprise:
+- create a Kubernetes cluster
+- install Weave GitOps Enterprise, using the `gitops-ee bootstrap` command
+- add your cluster to kubeconfig—which you'll get from Kubernetes—so that the kubeconfig correctly points to the management cluster
+- create a Git repository; in the instructions below, we refer to a `fleet-infra` repository
+- configure your Git client properly (if using GitHub, for example,
+then review their docs on [setting your username](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git#setting-your-git-username-for-every-repository-on-your-computer) and
+[your email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-email-address-for-every-repository-on-your-computer))
+- obtain a valid entitlement secret from Weaveworks and apply it to your cluster
+- install a compatible version of Flux onto your cluster; see below for how-to guidance
 
-### Prep Step: Create a Repository
-Create a new private GitHub repository and give it a name. We'll call our repo `fleet-infra`.
-
-Set up a Git client for your private repo. For GitHub, see their docs on [setting your username](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git#setting-your-git-username-for-every-repository-on-your-computer) and [setting your email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-email-address-for-every-repository-on-your-computer).
-
-[Cluster API](https://cluster-api.sigs.k8s.io/introduction.html) provides declarative APIs, controllers, and tooling to manage the lifecycle of Kubernetes clusters, across a large number of [infrastructure providers](https://cluster-api.sigs.k8s.io/reference/providers.html#infrastructure).
-The CAPI custom resource definitions are platform-independent as each provider implementation handles the creation of virtual machines,
-VPCs, networks, and other required infrastructure parts, enabling consistent and repeatable cluster deployments.
-
-The following example and steps reflect Flux’s architecture and operations. Go [here](https://fluxcd.io/docs/cmd/) for more detailed documentation about Flux.
-
-### 1. CAPA Setup
-
-Cluster API requires kubectl access to an existing Kubernetes cluster. For this example, configure kubectl to use the management cluster:
+As noted above, you receive your entitlements secret by contacting sales@weave.works. Use this command to apply it to the cluster:
 
 ```bash
-export KUBECONFIG=/path/to/kubeconfig
+kubectl apply -f entitlements.yaml
```

Comment:
This requires the `flux-system` namespace already exists. Could:

- Move it after the `flux bootstrap`
- Explicitly add a `kubectl create namespace flux-system`
- Maybe look into including the namespace def in the "entitlement file". Though that might intro other complications.

Reply:
@foot I took the first option

Reply:
> Do I also need to add the kubectl command?

No `flux bootstrap` will make sure the `flux-system` ns exists 👍

Reply:
Cool, thanks for the clarification